agent/structs/config_entry_test.go

```diff
@@ -1334,3 +1334,23 @@ func requireContainsLower(t *testing.T, haystack, needle string) {
 	t.Helper()
 	require.Contains(t, strings.ToLower(haystack), strings.ToLower(needle))
 }
+
+func TestConfigEntryQuery_CacheInfoKey(t *testing.T) {
+	assertCacheInfoKeyIsComplete(t, &ConfigEntryQuery{}, nil)
+}
+
+func TestServiceConfigRequest_CacheInfoKey(t *testing.T) {
+	ignoredFields := map[string]bool{
+		"QueryOptions": true,
+		// TODO: should this be included?
+		"UpstreamIDs": true,
```

Comment:
Note to self, looks like this is fixed in https://github.com/hashicorp/consul/pull/10240, so this ignore can be removed